test/v3/models/response/retrieveBalanceAndTransactions/RetrieveBalanceAndTransactionsResponseSpec.scala

```diff
@@ -51,7 +51,7 @@ class RetrieveBalanceAndTransactionsResponseSpec extends UnitSpec {
 
   "RetrieveBalanceAndTransactionsResponse.writes" should {
     "produce the expected JSON" in {
-      Json.toJson(response) shouldBe mtdResponseJson
+      Json.toJson(response) shouldBe mtdResponseWithPOARelevantAmountJson
```

Comment:
The feature switch is only temporary. Once the switch it turned on, there's a  risk that this naming will remain which could lead to questions and confusion. (e.g. "Why is this POA field special? Nothing in the code does anything differently with it. Hmm...!?")
So can we revert the naming of the various fixture fields (here and elsewhere) so that those _with_ the poa fields don't refer to it specifically (i.e. revert this one to `mtdResponseJson`) and those that temporarily _do not_ include the fields are named `...WithoutPoaRelevantAmount,,,` or whatever. It will make it easier to know what to clean up and what to leave.